orga/mirage/config.js

```diff
@@ -83,7 +83,24 @@ export default function () {
   this.patch('/memberships/:id');
 
   this.get('/organizations/:id/campaigns', (schema, request) => {
-    const results = schema.campaigns.all();
+    const { 'filter[creatorName]': creatorName, 'filter[name]': campaignName } = request.queryParams;
+    let results;
+    if (!creatorName && !campaignName) {
```

Comment:
J'ai l'impression qu'on refait ici le filtrage qui est fait côté api pour les tests d'acceptance. Est-ce que c'est possible avec les tests d'acceptance de vérifier qu'un GET a été fait côté avec la bonne url (de type `https://orga-pr3840.review.pix.fr/api/organizations/1/campaigns?filter%5Bname%5D=&filter%5Bstatus%5D=archived&filter%5BcreatorName%5D=&page%5Bnumber%5D=1&page%5Bsize%5D=25` ?

Reply:
je regarde cet exemple https://karolgalanciak.com/blog/2017/06/25/ember-tips-testing-outgoing-http-requests/

Reply:
Pas concluant, j'ai regardé la doc de mirage[ ici](https://miragejs.com/api/classes/server/) et modifié la config de mirage pour voir, sans grand succès

Reply:
'In general we recommend asserting against Mirage's database and your UI, as the specifics of your app's HTTP requests should be considered implementation details of the behavior you're actually interested in verifying. But there are certainly valid reasons to drop down and assert against HTTP data" > c'est même déconseillé https://miragejs.com/docs/testing/assertions/#asserting-against-handled-requests-and-responses

Reply:
Effectivement Yasmina m'a déjà alerté là dessus, mais je trouvais dommage et étrange qu'en déplaçant les composants que rien ne casse niveau test. C'est peut être overkill de tester ça mais c'est toujours mieux que n'avoir aucun test du tout.